pkg/etcd/etcd.go

```diff
@@ -1116,6 +1158,50 @@ func (e *ETCD) trackLearnerProgress(ctx context.Context, progress *learnerProgre
 	return e.setLearnerProgress(ctx, progress)
 }
 
+func (e *ETCD) setEtcdStatusCondition(node *v1.Node, promoted bool) (*v1.Node, error) {
+	var newCondition v1.NodeCondition
+	if promoted {
+		newCondition = v1.NodeCondition{
+			Type:               "EtcdIsVoter",
+			Status:             "True",
+			LastHeartbeatTime:  node.CreationTimestamp,
```

Comment:
Why are these hardcoded? Shouldn't the heartbeat time change every time the status is checked by the controller, and the transition time change whenever it changes? If these are updated automatically by Kubernetes, then just leave them blank here.

Reply:
They are hardcoded because they are `nil` by default, and to avoid the potential need for `nil` handling, I opted to go with default timestamps.